x-pack/metricbeat/module/statsd/server/server.go

```diff
@@ -62,26 +92,71 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 
 	processor := newMetricProcessor(config.TTL)
 
+	mappings, err := buildMappings(config.Mappings)
+	if err != nil {
+		return nil, fmt.Errorf("invalid mapping configuration for airflow module: %w", err)
```

Comment:
This is statsd generic code, don't mention airflow.
```suggestion
		return nil, fmt.Errorf("invalid mapping configuration: %w", err)
```